pkg/api/grpcServer.go

```diff
@@ -79,16 +79,21 @@ func (s *grpcServer) Check(legacyCtx legacyContext.Context, req *mixerpb.CheckRe
 
 	globalWordCount := int(req.GlobalWordCount)
 
-	glog.Info("Dispatching Preprocess")
+	if glog.V(1) {
```

Comment:
yes, in this case there was no expensive calculations/function calls
C++ has auto short circuit which is nice but this will do